### ISSUE TITLE
按照您之前帖子的操作好像不起作用

### ISSUE THREAD

**Issue by Issac304** [mdswyz/DMD#18]
output['s_l'].transpose(0, 1).contiguous().view(labels.size(0), -1)
                    output['s_v'].transpose(0, 1).contiguous().view(labels.size(0), -1)
                    output['s_a'].transpose(0, 1).contiguous().view(labels.size(0), -1)
                    output['c_l'].transpose(0, 1).contiguous().view(labels.size(0), -1)
                    output['c_v'].transpose(0, 1).contiguous().view(labels.size(0), -1)
                    output['c_a'].transpose(0, 1).contiguous().view(labels.size(0), -1)

                    cosine_similarity_s_c_l = self.cosine(output['s_l'], output['c_l'],
                                                          torch.tensor([-1]).cuda()).mean(0)
                    cosine_similarity_s_c_v = self.cosine(output['s_v'], output['c_v'],
                                                          torch.tensor([-1]).cuda()).mean(0)
                    cosine_similarity_s_c_a = self.cosine(output['s_a'], output['c_a'],
                                                          torch.tensor([-1]).cuda()).mean(0)
